### PR TITLE
Updated german files

### PR DESCRIPTION
**Pull request by Mr-Update**
Translation improved

### CODE REVIEW

File: _locales/de/messages.json [hunk 1]
```diff
@@ -716,15 +716,15 @@
 		"message": "E-Mail muss von der spezifizierten SDID signiert sein."
 	},
 	"ruletype.2.description": {
-		"message": "E-Mail muss nicht signiert sein. Falls Sie signiert ist, muss Sie von der spezifizierten SDID signiert sein."
+		"message": "E-Mail muss nicht signiert sein. Wenn Sie signiert ist, muss sie von der spezifizierten SDID signiert sein."
```
Comment (lieser) [inline _locales/de/messages.json:719]: Is there a reason to make the second `sie` lowercase, but keep the first one uppercase? Same question below for `ruletype.3.description`.

**Review by lieser — CHANGES_REQUESTED** [review]
Thanks a lot for the improvements. Looking good overall, just a few smaller comments.

File: _locales/de/messages.json [hunk 2]
```diff
@@ -244,27 +244,27 @@
...
 	"DKIM_SIGERROR_KEY_ILLFORMED_S": {
-		"message": "Service-Typ-Tag im DKIM-Schlüssel hat falsches Format"
+		"message": "Servicetyp-Tag im DKIM-Schlüssel hat falsches Format"
 	},
 	"DKIM_SIGERROR_KEY_TESTMODE": {
 		"message": "Die signierende Domain ist im DKIM-Testmodus"
 	},
 	"DKIM_SIGERROR_KEY_ILLFORMED_T": {
-		"message": "t-Tag im DKIM-Schlüssel hat falsches Format"
+		"message": "T-Tag im DKIM-Schlüssel hat falsches Format"
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply _locales/de/messages.json]: OK

File: _locales/de/messages.json [hunk 3]
```diff
@@ -375,29 +375,29 @@
 		"message": "Port:"
 	},
 	"options_dns.libunbound.nameserver": {
-		"message": "DNS-Server (Syntax (EBNF): IP{\";\"IP} )"
+		"message": "DNS-Server (EBNF-Syntax: IP{\";\"IP} )"
 	},
 	"options_dns.libunbound.usageWarning": {
-		"message": "Der libunbound Resolver benötigt eine extern installierte Bibliothek"
+		"message": "Der Libunbound-Resolver benötigt eine extern installierte Bibliothek!"
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply _locales/de/messages.json]: OK

File: _locales/de/messages.json [hunk 4]
```diff
@@ -479,35 +479,35 @@
...
-		"message": "Behandle nicht durch DNSSEC gesicherte DKIM-Schlüssel als"
+		"message": "Nicht durch DNSSEC gesicherte DKIM-Schlüssel behandeln als"
 	},
 	"options_error.algorithm.sign.rsa-sha1.treatAs": {
-		"message": "Behandle rsa-sha1-Signaturalgorithmus als"
+		"message": "RSA-SHA1-Signaturalgorithmus behandeln als"
 	},
 	"options_error.algorithm.rsa.weakKeyLength.treatAs": {
-		"message": "Behandle schwachen RSA-Schlüssel (<2048 bits) als"
+		"message": "Schwachen RSA-Schlüssel (< 2048 Bit) behandeln als"
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply _locales/de/messages.json]: OK

File: _locales/de/messages.json (hunk 1 above)
Comment (Mr-Update) [reply _locales/de/messages.json]: Sorry, I overlooked it.

File: _locales/de/messages.json [hunk 5]
```diff
@@ -412,29 +412,29 @@
...
-		"message": "Erlaube auch Subdomains der SDIDs"
+		"message": "Auch Subdomains der SDIDs erlauben"
 	},
 	"options_policy.signRules.error.wrong_sdid.asWarning": {
-		"message": "Behandle falsche SDID als Warnung statt als Fehler"
+		"message": "Falsche SDID als Warnung statt als Fehler behandeln"
 	},
 	"options_viewSigners": {
-		"message": "Signierregeln"
+		"message": "Signierregeln…"
```
[the remark this answers is not in the document]
Comment (Mr-Update) [reply _locales/de/messages.json:421]: … indicates that a dialog box opens when clicked.
The ellipsis points are actually common in all programs.

**Review by Mr-Update — COMMENTED** [review]
Modifications done

**Review by lieser — APPROVED** [review]

**Comment by lieser** [comment]
Thanks again for the improvements. I will update the texts on the ATN site when I make the next 4.1.1 release.

**Comment by lieser** [comment]
In 588cc029fa41eb187fb6ab51fed069d8c5f5adfd I fixed the ellipsis now being shown in some titles.
I also added it to a few more buttons opening new windows, and adapted the other translations.

**Pull request merged**